src/Microsoft.VisualStudio.ProjectSystem.Managed/IO/SimpleFileWatcher.cs

```diff
@@ -1,23 +1,31 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
+using Microsoft.VisualStudio.FileWatch;
+using Microsoft.VisualStudio.ProjectSystem;
+
 namespace Microsoft.VisualStudio.IO;
 
 /// <summary>
 /// Simple wrapper around the FileSystemWatcher.
 /// </summary>
-internal sealed class SimpleFileWatcher : IDisposable
+[Export(typeof(IFileWatcherService))]
+[Order(Order.Lowest)]
```

Comment:
@drewnoakes 

How is `Order` works here,, If I implement another `IFileWatcherService` with a higher order, will that be used to create `LaunchSettingProvider` instead?

Reply:
`[Order]` is metadata that a MEF consumer can use to differentiate between imports when there are multiple available. In this PR I don't see any `ImportsMany` (or similar constructs like `OrderPrecedenceImportCollection`), so I don't think `Order` is needed.